clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/ObjectStorageGateway.java

```diff
@@ -877,14 +898,23 @@ public PostObjectResponseType postObject(PostObjectType request) throws S3Except
    */
   @Override
   public DeleteObjectResponseType deleteObject(final DeleteObjectType request) throws S3Exception {
-    ObjectEntity objectEntity;
+    ObjectEntity objectEntity = null;
+    Bucket bucket = null;
     try {
+      //LPT Bug! Throws exception on anonymous access to an object
+      //bucket = getBucketAndCheckAuthorization(request);
+      //LPT The fix:
```

Comment:
EUCA-12174 still needs to be fixed. For now, the code is reverted to its original behavior, which will throw an exception on anonymous write access.